ui/src/helpers/get-placeholder.js

```diff
@@ -0,0 +1,32 @@
+export default function (type, defaultPlaceholder = 'Digite aqui...') {
```

Comment:
Eu acho que o nome da função faz mais sentido se for `getPlaceholderByMask`

Reply:
Será? Pois pode ser por `mask` ou por `type`...

Reply:
![Screenshot 2023-09-25 at 11 36 14](https://github.com/bildvitta/asteroid/assets/19765074/d5776d7f-90e3-431a-8ce4-76ddf96ab686)